src/collector/top_collector.rs

```diff
@@ -175,6 +180,26 @@ impl<T: PartialOrd + Clone> TopSegmentCollector<T> {
     }
 }
 
+fn stable_sort<T, D>(a: &ComparableDoc<T, D>, b: &ComparableDoc<T, D>) -> Ordering
```

Comment:
it's not a `stable_sort` so it should not be called stable sort.

Reply:
(this code doesn't exist anymore if we go for V2)